apps/test/unit/gamelab/spritelab/spriteCommandsTest.js

```diff
@@ -0,0 +1,157 @@
+    spriteUtils.addSprite(sprite1);
+    spriteUtils.addSprite(sprite2);
+    spriteUtils.addSprite(sprite3);
+
+    expect(commands.countByAnimation('a')).to.equal(2);
+    expect(commands.countByAnimation('b')).to.equal(1);
+    expect(commands.countByAnimation('c')).to.equal(0);
+  });
+
+  it('destroy single sprite', () => {
+    let sprite1 = createSprite();
+    let sprite2 = createSprite();
+    let id1 = spriteUtils.addSprite(sprite1);
+    let id2 = spriteUtils.addSprite(sprite2);
+    expect(spriteUtils.getSpriteIdsInUse()).to.have.members([id1, id2]);
+
+    commands.destroy(id1);
+    expect(spriteUtils.getSpriteIdsInUse()).to.have.members([id2]);
+
+    commands.destroy(id2);
+    expect(spriteUtils.getSpriteIdsInUse()).to.have.members([]);
+  });
+
+  it('destroy animation group', () => {
+    let sprite1 = createSprite();
+    let sprite2 = createSprite();
+    let sprite3 = createSprite();
+    sprite1.addAnimation('a', animation);
+    sprite2.addAnimation('b', animation);
+    sprite3.addAnimation('a', animation);
+    spriteUtils.addSprite(sprite1);
+    let id2 = spriteUtils.addSprite(sprite2);
+    spriteUtils.addSprite(sprite3);
+
+    commands.destroy('a');
+
+    expect(spriteUtils.getSpriteIdsInUse()).to.have.members([id2]);
+  });
+
+  it('getProp for single sprite', () => {
+    let sprite = createSprite();
+    sprite.position.x = 123;
+    sprite.position.y = 321;
+    sprite.addAnimation('label', animation);
+    sprite.anotherProp = 'value';
+    let id = spriteUtils.addSprite(sprite);
+    expect(commands.getProp(id, 'x')).to.equal(123);
+    expect(commands.getProp(id, 'y')).to.equal(79);
+    expect(commands.getProp(id, 'costume')).to.equal('label');
+    expect(commands.getProp(id, 'anotherProp')).to.equal('value');
+  });
+
+  it('getProp for animation group uses the first sprite in the group', () => {
+    let sprite1 = createSprite();
+    sprite1.addAnimation('label', animation);
+    sprite1.position.x = 123;
+    spriteUtils.addSprite(sprite1);
+
+    let sprite2 = createSprite();
+    sprite2.addAnimation('label', animation);
+    sprite2.position.x = 321;
+    spriteUtils.addSprite(sprite2);
+
+    expect(commands.getProp('label', 'x')).to.equal(123);
+  });
+
+  describe('makeSprite', () => {
+    let makeSprite;
+    beforeEach(function() {
+      makeSprite = commands.makeSprite.bind(gameLabP5.p5);
+    });
+
+    it('returns an id', () => {
+      expect(makeSprite()).to.equal(0);
+      expect(makeSprite()).to.equal(1);
+      expect(makeSprite()).to.equal(2);
+    });
+
+    it('location defaults to (200,200)', () => {
+      let id = makeSprite();
+      expect(commands.getProp(id, 'x')).to.equal(200);
+      expect(commands.getProp(id, 'y')).to.equal(200);
+    });
+
+    it('location picker works', () => {
+      let id = makeSprite(undefined, {x: 123, y: 321});
+      expect(commands.getProp(id, 'x')).to.equal(123);
+      expect(commands.getProp(id, 'y')).to.equal(400 - 321);
```

Comment:
the 400-y format you have here is clearer than what you have above (i.e. line 73 as an example). Use this format there as well.